genesis/model/resolver/context.go

```diff
@@ -51,19 +54,24 @@ func (r *contextResolver) GetObject(ref *object.Reference, classID string) (obje
 	}
 
 	for proxy.GetClassID() == class.ReferenceID {
-		contextHolderWithChildInterface, ok := contextHolder.(object.Child)
-		if !ok {
+		contextHolderWithChildInterface, isOk := contextHolder.(object.Child)
```

Comment:
`ok` was better name

Reply:
declaration of ```ok``` shadows declaration at line 40 (in this function but outside of cycle)

Reply:
`isChild` then ;)